[Alerting][Docs] Documentation for action variables. (#92417)
elastic/kibana: docs/user/alerting/alert-types.asciidoc

[role="screenshot"]
image::images/alert-types-index-threshold-preview.png[Five clauses define the condition to detect]

[float]
==== Action variables

When the index threshold alert condition is met, the following variables are available to use inside each action:

> `context.title`:: A pre-constructed title for the alert. Example: `alert kibana sites - high egress met threshold`.

Reviewer: ```suggestion
`context.title`:: A preconstructed title for the alert. Example: `alert kibana sites - high egress met threshold`.
```

> `context.message`:: A pre-constructed message for the alert. +

Reviewer: ```suggestion
`context.message`:: A preconstructed message for the alert. Example:+
```

> Example: `alert 'kibana sites - high egress' is active for group 'threshold met':` +

Reviewer: ```suggestion
`alert 'kibana sites - high egress' is active for group 'threshold met':` +
```

`- Value: 42` +
`- Conditions Met: count greater than 4 over 5m` +
`- Timestamp: 2020-01-01T00:00:00.000Z`

`context.group`:: The name of the action group associated with the threshold condition. Example: `threshold met`.

> `context.date`:: The date the alert met the threshold condition in ISO format. Example: `2020-01-01T00:00:00.000Z`.

Reviewer: ```suggestion
`context.date`:: The date, in ISO format, that the alert met the threshold condition. Example: `2020-01-01T00:00:00.000Z`.
```

Threshold:: This clause defines a threshold value and a comparison operator  (`is above`, `is above or equals`, `is below`, `is below or equals`, or `is between`). The number of documents that match the specified query is compared to this threshold.
Time window:: This clause determines how far back to search for documents, using the *time field* set in the *index* clause. Generally this value should be set to a value higher than the *check every* value in the <<defining-alerts-general-details, general alert details>>, to avoid gaps in detection. 

[float]
==== Action variables

When the ES query alert condition is met, the following variables are available to use inside each action:

> `context.title`:: A pre-constructed title for the alert. Example: `alert term match alert query matched`.

Reviewer: ```suggestion
`context.title`:: A preconstructed title for the alert. Example: `alert term match alert query matched`.
```

> `context.message`:: A pre-constructed message for the alert. +

Reviewer: ```suggestion
`context.message`:: A pre-constructed message for the alert. Example:+
```

> Example: `alert 'term match alert' is active:` +

Reviewer: ```suggestion
`alert 'term match alert' is active:` +
```

`- Value: 42` +
`- Conditions Met: count greater than 4 over 5m` +
`- Timestamp: 2020-01-01T00:00:00.000Z`

`context.group`:: The name of the action group associated with the condition. Example: `query matched`.

> `context.date`:: The date the alert met the condition in ISO format. Example: `2020-01-01T00:00:00.000Z`.

Reviewer: ```suggestion
`context.date`:: The date, in ISO format, that the alert met the condition. Example: `2020-01-01T00:00:00.000Z`.
```

> `context.value`:: The value for the alert that met the condition.

Reviewer: The value for > The value of?
